misc/test-stubgenc.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# This script is expected to be run from root of the mypy repo
+
+# Install dependencies, demo project and mypy
+python -m pip install -r test-requirements.txt
+python -m pip install pybind11 numpy pybind11-mypy-demo
+python -m pip install pybind11-mypy-demo==0.0.1
```

Comment:
Why do you first install pybind11-mypy-demo without a specific version? Also, it might be better to pin the versions of pybind11 and numpy too.

Reply:
That's a typo, thanks for spotting it!

In fact, `pybind11` and `numpy` can be removed. `pybind11`  is required to build from source only. `numpy` tests are not there (yet), so can be omitted as well. 

Future note (for myself): `numpy` should be listed in `pybind11-mypy-demo` runtime requirements, not here)